Makefile

```diff
@@ -42,17 +42,13 @@ $(FC_TEST_DATA_PATH)/ltag \
 $(FC_TEST_BIN_PATH)/ptp \
 $(FC_TEST_BIN_PATH)/host-local \
 $(FC_TEST_BIN_PATH)/static \
-$(FC_TEST_BIN_PATH)/tc-redirect-tap
-
-# Enable pulling of artifacts from S3 instead of building
-# TODO: https://github.com/firecracker-microvm/firecracker-go-sdk/issues/418
-ifeq ($(GID), 0)
-testdata_objects += $(FC_TEST_DATA_PATH)/root-drive-with-ssh.img $(FC_TEST_DATA_PATH)/root-drive-ssh-key
-endif
+$(FC_TEST_BIN_PATH)/tc-redirect-tap \
+$(FC_TEST_DATA_PATH)/root-drive-with-ssh.img \
+$(FC_TEST_DATA_PATH)/root-drive-ssh-key
```

Comment:
nit: alphabetize the order of test object. the new targets moved after L41?

Reply:
Updated.